src/ck3/data/religions.rs

```diff
@@ -69,7 +71,44 @@ impl DbKind for Religion {
         vd.field_item("family", Item::ReligionFamily);
 
         vd.req_field("doctrine");
-        vd.multi_field_item("doctrine", Item::Doctrine);
+        let mut categories: FnvHashMap<&str, Vec<Token>> = FnvHashMap::default();
+        vd.multi_field_validated_value("doctrine", |_, mut vd| {
+            vd.item(Item::Doctrine);
+            if let Some(category) = data.doctrines.category(vd.value().as_str()) {
+                let doctrine = vd.value();
+                if let Some(seen) = categories.get_mut(category.as_str()) {
+                    let picks = data.doctrines.number_of_picks(category.as_str());
+                    #[allow(clippy::cast_possible_wrap)]
+                    if picks == 1 {
+                        // SAFETY: we never push empty vecs into this hash
+                        let other_doctrine = &seen[0];
+                        if doctrine == other_doctrine {
+                            let msg = format!("duplicate doctrine {doctrine}");
```

Comment:
I don't think duplicate doctrine is being checked for when picks > 1?

Reply:
Yeah it's worth checking it there too